.github/workflows/test_installer.yml

```diff
@@ -1,75 +1,181 @@
-name: Check Installer
+name: Check Installer (scheduled)
 
 on:
   push:
   pull_request:
   workflow_dispatch:
+  schedule:
+    - cron: '30 5 * * 1'
 
 jobs:
-  install-linux:
+  install-mss-on-linux-from-scratch:
     runs-on: ubuntu-latest
     
     defaults:
       run:
         shell: bash
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Install conda+MSS
+      - name: Install mambaforge+MSS
         run: |
-          sudo rm -rf $(which conda)
+          rm -rf $(which mamba)
           cd $GITHUB_WORKSPACE
           chmod +x LinuxMac.sh
           ./LinuxMac.sh -a
       - name: Test MSS installed
         run: |
-          . /home/runner/miniconda3/bin/activate mssenv
-          msui --version | grep "Mission Support System (MSS)"
+          /home/runner/mambaforge/envs/mssenv/bin/msui --version | grep "Mission Support System (MSS)"
```

Comment:
Does this activate the environment? We had some issues in th past, with, e.g. pyproj and un-activated environemnts.